juan-pablo-lopez/Tarea 3/js/spongeMeme.js

```diff
@@ -0,0 +1,12 @@
+function spongeMeme(sentence){
+    let result = "";
+    for(let i=0; i < sentence.length; i++){
```

Comment:
@juanpablo-lopez01 , use modern approaches for array operations like forEach, map, reduce, in fact this one could be a one liner like oddOrEven solution